File: TwoSum/TwoSum.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
#include <stdio.h>

int* twoSum(int* nums, int numsSize, int target, int* returnSize){
    int i, j, sum = 0;
    int *ans = (int*)malloc(2 * sizeof(int));

    for(i; i < numsSize; i++){
        for(j=i+1; j < numsSize; j++){
            sum = nums[i] + nums[j];
            if (sum == target){
                ans[0] = i;
                ans[1] = j;
                *returnSize = 2;
                return ans;
            }
        }
    }
    *returnSize = 0;
    return NULL;
}

File: TwoSum/twoSum.py
"""this gives the sum of two numbers that will equate to target

Returns:
    _type_: integer
"""
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        lens = len(nums)
        for i in range(0,lens):
            for j in range(i+1, lens):
                sum = nums[i] + nums[j]
                if(sum == target):
                    ans = []
                    ans.append(i)
                    ans.append(j)
                    return ans
  